data/index.js

```diff
@@ -10,7 +10,7 @@ db.once('open', () => console.log('db connected'));
 
 const earningsSchema = new mongoose.Schema({
   ticker: { type: String, required: true, unique: true },
-  name: { type: String, required: true, unique: true },
+  name: String,
```

Comment:
Did you remove `required` and `unique` from `name` because the ticker is more like an index?

Reply:
We can disregard this since this was set up for MongoDB